### PR TITLE
Remove oauth creation

### PR DESCRIPTION
**Pull request by BerkeleyTrue**
ref #8167

### CODE REVIEW

File: common/models/User-Identity.js [hunk 1]
```diff
@@ -40,96 +28,93 @@ export default function(UserIdent) {
...
 
-      if (!userObj.email && !options.emailOptional) {
-        process.nextTick(function() {
-          return cb('email is missing from the user profile');
-        });
-      }
+        const userObj = options.profileToUser(provider, profile, options);
+        if (getSocialProvider(provider) !== 'github') {
+          return process.nextTick(() => cb(
+            new Error(
```
Comment (raisedadead) [inline common/models/User-Identity.js]: Not sure if I am doing something horribly wrong while testing this, but it throws an exception instead shouldn't we have a flash error?

Comment (BerkeleyTrue) [inline common/models/User-Identity.js]: Yes, I don't think there is a built in way to show a flash message through this API. This PR will rely on yours

Comment (raisedadead) [inline common/models/User-Identity.js]: Thanks for the clarification, I'll ensure that my PR is addresses this.

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **PERFECTO.** :sparkles: :sparkles: 

Thanks @BerkeleyTrue !

**Pull request merged**